templates/signatures_job.py.tpl

```diff
@@ -17,21 +17,21 @@ job.init(args['JOB_NAME'], args)
 
```

Comment:
These changes are not related to the Lambdas but I've been using this branch while troubleshooting the Glue connection issue